include/IDescriptorSet.h

```diff
@@ -69,7 +69,7 @@ class IDescriptorSet : public virtual IVulkanPrebaked
                         texture->drop();
                 }
 
-                IVirtualTexture* texture;
+                IRenderableVirtualTexture* texture;
                 STextureSamplingParams sampler;
         };
         CombinedImageSampler texturesAndSamplers[_IRR_MATERIAL_MAX_TEXTURES_];
```

Comment:
don't set a max anymore, on anything

Reply:
let us query

Reply:
same for the other objects.

Remember we want to use `EXT_descriptor_indexing` in the future.

Reply:
>don't set a max anymore, on anything

what about vertex attrib buffer bindings (described by params in pipeline) in meshbuffer? Should there still be limit of 16 and relation binding_num=attrib_num?

Reply:
yes, vertex attrib buffer bindings are not descriptors. Neither are subpass inputs (really).

Other bindings with `EXT_descriptor_indexing` you could have 512k textures bound at once.

Reply:
so there still limit of 16 attrib buffer bindings? then array index cannot denote binding num?

Reply:
https://github.com/buildaworldnet/IrrlichtBAW/pull/358#discussion_r324440409
until now we had convention that binding_num=location (attrib num). so why would we drop this if nothing changes here?

Reply:
>until now we had convention that binding_num=location (attrib num). so why would we drop this if nothing changes here? 

This will no longer be our contention

Reply:
> so there still limit of 16 attrib buffer bindings? then why array index cannot denote binding num?

Faster buffer rebinding, probably better hardware cache usage.

Reply:
@Crisspl  ?

Reply:
resolved